### ISSUE TITLE
Mifare-Classic-1k---Money-Read-HELP-ME

### ISSUE THREAD

**Issue by trustedman** [trustedman/Mifare-Classic-1k---Money-Read-HELP-ME#1]
Well guys, how are you?

I need help to find the exact value of money for this Mifare Classic 1k card. I already discovered the keys of the 15 sectors and to my surprise there are 2 lines of values ​​that are in sector 0.

I don't just want to clone, I want to learn how to find the exact value as I said and be able to make regridges.

![WhatsApp Image 2020-11-22 at 09 59 50](https://user-images.githubusercontent.com/56659345/99977067-f8512d00-2d82-11eb-88e6-7ea2016cf951.jpeg)

![WhatsApp Image 2020-11-23 at 12 07 13](https://user-images.githubusercontent.com/56659345/99982552-7ca6ae80-2d89-11eb-8390-a4b74797235f.jpeg)

**Comment by ikarus23** [comment]
Well, it is yellow. Yellow means "Value Block". you can decode value blocks by pressing the menu (3 dots top right) and select "Value Blocks as Int."

**Comment by ikarus23** [comment]
Read the MIFARE Classic datasheet if you want to know how value blocks work.

**Comment by trustedman** [comment]
Thank u for helpe me.. and sorry for "spam" I will delete the other issue. 

Yes, and how this number ( 2147482589 ) will help me to get the exact value ?? 
In this card have $28,95 in money 

Thank you for the tip, I will read about. 

![WhatsApp Image 2020-11-24 at 16 13 15](https://user-images.githubusercontent.com/56659345/100141053-8bb45c00-2e70-11eb-9305-7c811e1fcdbb.jpeg)

**Comment by ikarus23** [comment]
Interesting. This looks more like counter. I don't think the money is stored there. Maybe it is in another sector?
28,95 = 2895 = 0x0B4F. Have you seen the value "0B4F" somewhere?

**Comment by trustedman** [comment]
Yes, I agree with u. Look this 2 Dumps and compare.. 

+Sector: 0 - $33,90
E5E80AC2C50804006263646566676869
**DEFBFF7F21040080**DEFBFF7F00FF00FF - Value block
**DFFBFF7F20040080D**FFBFF7F00FF00FF - Value block
7B296F353C6BE69871CBC592B3A32380

After using $4,95 

+Sector: 0 - $28,95
E5E80AC2C50804006263646566676869
**DDFBFF7F22040080**DDFBFF7F00FF00FF - Value block
**DEFBFF7F21040080**DEFBFF7F00FF00FF - Value block
7B296F353C6BE69871CBC592B3A32380

I dont found 0B4F :( 

All the $28,95 dump.. 

+Sector: 0
E5E80AC2C50804006263646566676869
DDFBFF7F22040080DDFBFF7F00FF00FF
DEFBFF7F21040080DEFBFF7F00FF00FF
7B296F353C6BE69871CBC592B3A32380

+Sector: 1
F1FF020040D6E87B0000000000000000
00000000000000000000C0110437C2C0
F6AB243B000000000000949D1184FDB1
3FA7217EC575F0FF00FEB17A8968D4EC

+Sector: 2
02C80CB30B0D7436278007800720C3EC
42C82C08407400000040EFFBF4FFFFFF
FFFFFFFFFFFFFFFFFFFF6C73F7D352B4
3FA7217EC575F0FF00FEE1255CEAFDEA

+Sector: 3
00000000000000000000000000000000
00000000000000000000000000000000
00000000000000000000000000000000
F630FC46B603F0FF00FE63508EB64603

+Sector: 4
120000000000000000B05F3B3A3100D0
C5FE0482790C973340033800C3ECC200
C0BDF0900C1604000000005E2C087B05
37DC29F80D81F0FF00FEC6179F4A80B5

+Sector: 5
220000000000000000B05F3B3A4100D0
84380182A70C003200000080A97C9700
8DBE00DC0B0B00000000003EBB7C5352
CBD70D8CA728F0FF00FEB57FC4A4B094

+Sector: 6
00000000000000000000000000000000
00000000000000000000000000000000
00000000000000000000000000000000
F630FC46B606F0FF00FE63508EB64606

+Sector: 7
00000000000000000000000000000000
00000000000000000000000000000000
00000000000000000000000000000000
F630FC46B607F0FF00FE63508EB64607

+Sector: 8
00000000000000000000000000000000
00000000000000000000000000000000
00000000000000000000000000000000
F630FC46B608F0FF00FE63508EB64608

+Sector: 9
00000000000000000000000000000000
00000000000000000000000000000000
00000000000000000000000000000000
F630FC46B609F0FF00FE63508EB64609

+Sector: 10
00000000000000000000000000000000
00000000000000000000000000000000
00000000000000000000000000000000
F630FC46B60AF0FF00FE63508EB6460A

+Sector: 11
02C80A736F0C91BE2440064006A0C2DC
1BC82A08407400000040F0FBF4FFFFFF
FFFFFFFFFFFFFFFFFFFF96208380DF5B
3FA7217EC575F0FF00FEE1255CEAFDEA

+Sector: 12
00000000000000000000000000000000
00000000000000000000000000000000
00000000000000000000000000000000
F630FC46B60CF0FF00FE63508EB6460C

+Sector: 13
120000000000000000B05F3B3A3100D0
C5FE0482790C973340033880C2DC9B00
C0BD00AE0C1504000000009B1468BC7B
37DC29F80D81F0FF00FEC6179F4A80B5

+Sector: 14
00000000000000000000000000000000
00000000000000000000000000000000
00000000000000000000000000000000
F630FC46B60EF0FF00FE63508EB6460E

+Sector: 15
220000000000000000B05F3B3A4100D0
84380182A70C003200000080A97C9700
8DBE00DC0B0B00000000003EBB7C5352
CBD70D8CA728F0FF00FEB57FC4A4B094

Only the sectors below have a change when I use the balance

![sector 0](https://user-images.githubusercontent.com/56659345/100144298-5bbb8780-2e75-11eb-925f-d43505cbc1af.jpeg)
![sector 2](https://user-images.githubusercontent.com/56659345/100144304-5fe7a500-2e75-11eb-9ea3-ed0efc482fea.jpeg)
![sector 4](https://user-images.githubusercontent.com/56659345/100144316-6249ff00-2e75-11eb-99b6-d15beb4ddbef.jpeg)
![sector 11](https://user-images.githubusercontent.com/56659345/100144323-637b2c00-2e75-11eb-83a3-081529d68400.jpeg)
![sector 13](https://user-images.githubusercontent.com/56659345/100144332-6544ef80-2e75-11eb-8c90-ef32f288ce15.jpeg)

**Comment by ikarus23** [comment]
Wow. A lot of stuff changed. I never saw a system with so much changes just by one purchase. I don't see the balance. I think you have to collect more data. Maybe multiple small purchases within a small time frame (with a dump after each purchase).

**Comment by trustedman** [comment]
Yes man, a lot changes :( I never saw the balance too, in hex or decimal.. 
But ok, thank you help me. I will try make 2 charges and dumps, for me to see what changes will occur after the balance is recharged on the card and do what you said too <3
